docs/user-guide/services.md

```diff
@@ -355,23 +355,32 @@ they desire, but leaves them freedom to do discovery in their own way.
 Applications can still use a self-registration pattern and adapters for other
 discovery systems could easily be built upon this API.
 
-## External services
+## Publishing services - service types
 
 For some parts of your application (e.g. frontends) you may want to expose a
 Service onto an external (outside of your cluster, maybe public internet) IP
-address.  Kubernetes supports two ways of doing this: `NodePort`s and
-`LoadBalancer`s.
+address, other services should be visible only from inside of the cluster.
 
-Every `Service` has a `type` field which defines how the `Service` can be
-accessed.  Valid values for this field are:
 
-   * `ClusterIP`: use a cluster-internal IP only - this is the default and is
-     discussed above
-   * `NodePort`: use a cluster IP, but also expose the service on a port on each
-     node of the cluster (the same port on each node)
-   * `LoadBalancer`: use a ClusterIP and a NodePort, but also ask the cloud
-     provider for a load balancer which forwards to the `Service`
+Kubernetes `ServiceTypes` allows you to specify what kind of service you want.
```

Comment:
grammar and styling: "`ServiceType`s allow ..." the type name is ServiceType (singular), but since you pluralize it, change "allows" to "allow".  Yay English.